node/service/src/relay_chain_selection.rs

```diff
@@ -444,8 +444,7 @@ where
 			let (subchain_number, subchain_head) = rx
 				.await
 				.map_err(Error::OverseerDisconnected)
-				.map_err(|e| ConsensusError::Other(Box::new(e)))?
-				.unwrap_or_else(|| (subchain_number, subchain_head));
```

Comment:
Here we would previously use subchain_number/subchain_head in case we get `None` from the dispute coordinator.

Reply:
Got it